WalletWasabi.Fluent/ViewModels/HelpAndSupport/AboutViewModel.cs

```diff
@@ -95,24 +97,25 @@ public AboutViewModel()
 		OpenBrowserCommand = ReactiveCommand.CreateFromTask<string>(IoHelpers.OpenBrowserAsync);
 
 		AboutAdvancedInfoDialogCommand = ReactiveCommand.CreateFromTask(
-			execute: async () => await NavigateDialogAsync(new AboutAdvancedInfoViewModel(), NavigationTarget.CompactDialogScreen));
+			execute: async () =>
+				await NavigateDialogAsync(new AboutAdvancedInfoViewModel(), NavigationTarget.CompactDialogScreen));
 
 		OpenBrowserCommand = ReactiveCommand.CreateFromTask<string>(
 			async (link) =>
 				await IoHelpers.OpenBrowserAsync(link));
 
 		CopyLinkCommand = ReactiveCommand.CreateFromTask<string>(
 			async (link) =>
+			{
+				if (Application.Current is {Clipboard: { } clipboard})
 				{
-					if (Application.Current is { Clipboard: { } clipboard })
-					{
-						await clipboard.SetTextAsync(link);
-					}
-				});
+					await clipboard.SetTextAsync(link);
+				}
+			});
 
 		NextCommand = CancelCommand;
 
-		SetupCancel(enableCancel: false, enableCancelOnEscape: true, enableCancelOnPressed: true);
+		SetupCancel(enableCancel: false, enableCancelOnEscape: false, enableCancelOnPressed: false);
```

Comment:
Why was this required?

Reply:
woops thanks, wasnt supposed to change that